tech-far-hub/src/components/resouces-card.tsx

```diff
@@ -12,9 +13,13 @@ interface IResourceNode {
 }
 
 const ResouceCard = ({ node, width = 3 }: { node: IResourceNode; width?: ColumnSizes }): JSX.Element => {
-  if (node.html && node.frontmatter && node.frontmatter.href) {
+  if (node.html && node.frontmatter && node.frontmatter.href && node.frontmatter.heading) {
     return (
-      <Card headerFirst gridLayout={{ tablet: { col: width } }}>
+      <Card
+        headerFirst
+        gridLayout={{ tablet: { col: width } }}
+        key={`resource-${_.snakeCase(node.frontmatter.heading)}`}
```

Comment:
Good ol' lodash, forgot about this lib!

Reply:
They use it in some docs for gatsby and i was like "Oh right i love that library!"